.github/workflows/publish-docker.yml

```
@@ -10,11 +10,16 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - uses: docker/build-push-action@v2
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: excalidraw/excalidraw
-          tag_with_ref: true
-          tag_with_sha: true
+          password: ${{ secrets.DOCKER_TOKEN }}
```

```suggestion
          password: ${{ secrets.DOCKER_PASSWORD }}
```
use older token, we will rename it soon